File: spring-cloud-starter/src/main/java/com/jmsoftware/maf/springcloudstarter/quartz/entity/QuartzJobConfigurationExcel.java
package com.jmsoftware.maf.springcloudstarter.quartz.entity;

import lombok.Data;

import javax.validation.constraints.NotBlank;
import java.time.LocalDateTime;

/**
 * Description: QuartzJobConfigurationExcel, change description here.
 *
 * @author Johnny Miller (锺俊), email: johnnysviva@outlook.com, date: 9/27/2021 11:26 AM
 **/
@Data
public class QuartzJobConfigurationExcel {
    @NotBlank
    private String name;
    private String group;
    private String serviceName;
    @NotBlank
    private String invokeTarget;
    @NotBlank
    private String cronExpression;
    private Byte misfirePolicy;
    private Byte concurrent;
    @NotBlank
    private String description;
    private Byte status;
    private String createdBy;
    private LocalDateTime createdTime;
    private String modifiedBy;
    private LocalDateTime modifiedTime;
}

File: spring-cloud-starter/src/main/kotlin/com/jmsoftware/maf/springcloudstarter/quartz/entity/QuartzJobConfigurationExcel.kt
package com.jmsoftware.maf.springcloudstarter.quartz.entity

import jakarta.validation.constraints.NotBlank
import java.time.LocalDateTime

/**
 * # QuartzJobConfigurationExcel
 *
 * Description: QuartzJobConfigurationExcel, change description here.
 *
 * @author Johnny Miller (锺俊), e-mail: johnnysviva@outlook.com, date: 4/14/22 10:39 PM
 */
class QuartzJobConfigurationExcel {
    var name: @NotBlank String? = null
    var group: String? = null
    var serviceName: String? = null
    var invokeTarget: @NotBlank String? = null
    var cronExpression: @NotBlank String? = null
    var misfirePolicy: Byte? = null
    var concurrent: Byte? = null
    var description: @NotBlank String? = null
    var status: Byte? = null
    var createdBy: String? = null
    var createdTime: LocalDateTime? = null
    var modifiedBy: String? = null
    var modifiedTime: LocalDateTime? = null
}
